CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -30,6 +30,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ### Dependencies
 - Compile binaries using [go 1.20.10](https://github.com/etcd-io/etcd/pull/16745).
+- Upgrade gRPC to 1.58.3 in https://github.com/etcd-io/etcd/pull/16625, https://github.com/etcd-io/etcd/pull/16781 and https://github.com/etcd-io/etcd/pull/16790. gRPC server will reject requests with connection header (refer to https://github.com/grpc/grpc-go/pull/4803).
```

Comment:
```suggestion
- Upgrade gRPC to 1.58.3 in https://github.com/etcd-io/etcd/pull/16625, https://github.com/etcd-io/etcd/pull/16781 and https://github.com/etcd-io/etcd/pull/16790. Note that gRPC server will reject requests with connection header (refer to https://github.com/grpc/grpc-go/pull/4803).
```

Reply:
Sorry, I should raise this comment along with previous comment.

Reply:
No worry. I am wondering if CHANGELOG has a breaking behavior change section but it could also be appended to the release notes.